### ISSUE TITLE
Could we avoid this prompt on the Desktop side?

### ISSUE THREAD

**Issue by TylerLeonhardt** [microsoft/vscode#162099]
Testing #161872

Today, we get this prompt on the web side and on the Desktop side when we Continue on...
![image](https://user-images.githubusercontent.com/2644648/192668518-3ca73846-89db-424f-bd3f-28034231823f.png)

Because we're in the process of "Continuing On..." could you maybe tell Desktop what auth provider was used to log in to Edit Sessions in the web? That way I don't have to remember which account I used when I signed up for Edit Sessions in the web.

**Comment by joyceerhl** [comment]
Yeah this would address https://github.com/microsoft/vscode/issues/162025

We could pass the auth provider and account name in more protocol url query params, but I start to worry we might hit the url limit for browsers...
